ui/src/logs/containers/LogsPage.tsx

```diff
@@ -358,9 +378,14 @@ class LogsPage extends PureComponent<Props, State> {
     )
   }
 
-  private handleUpdateSeverityLevels = (levels: SeverityLevel[]) => {
-    // Todo: Handle saving of these new severity colors here
-    levels = levels
+  private handleUpdateSeverityLevels = async (
+    severityLevelColors: SeverityLevelColor[]
+  ) => {
```

Comment:
A few of the handlers  are annotated with `void`  return, but some of them aren't

Reply:
ill make sure there are types on those returns. nice catch thanks